### PR TITLE
Feature/ivashchenko#317navbar mobile friendly

### PR DESCRIPTION
**Pull request by DenysIvashchenko**
need to complete the functionality for the navigation link "Фільтри" and button "СКИНУТИ" after mobileView styling  the component result.component.html .

toDo

### CODE REVIEW

File: src/app/header/header.component.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
   line-height: none;
 }
 
-.poinrer{
+.pointer{
```
Comment (dmitryy90) [inline src/app/header/header.component.scss:10]: is it used somewhere?

File: src/app/header/header.component.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit,HostListener } from '@angular/core';
```
Comment (dmitryy90) [inline src/app/header/header.component.ts:1]: indentations

File: src/app/header/header.component.ts [hunk 3]
```diff
@@ -33,9 +40,23 @@ export class HeaderComponent implements OnInit {
     private router: Router
   ) { }
 
+  /**
+   * @param event global variable window
+   * method defined window.width and assign MobileView: boolean     
+   */
+  isWindowMobile(event: any): void {
+    (event.innerWidth <= 750) ?  this.MobileView = true : this.MobileView = false;
```
Comment (dmitryy90) [inline src/app/header/header.component.ts:48]: this.MobileView = event.innerWidth <= 750;

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/header/header.component.scss (hunk 1 above)
Comment (DenysIvashchenko) [reply src/app/header/header.component.scss:10]: yes,
pointer

 I saw in header.component.html

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**